### PR TITLE
fix(file-uploader): fixes multiple tabbable elements bug and adds keyboard navigation to label

### LINKED ISSUE #1351: button has incorrect aria and is not keyboard accessible
## Environment
Chrome Version 70.0.3538.67 (Official Build) (64-bit)
macOS Mojave 10/Windows 10

## Violations:
- **The WAI-ARIA role(s) and/or attribute(s) button are not valid for the element label. See rule help for guidance.**

screen shot 2018-10-26 at 3 49 24 pm

- **the add file button cannot be thrown using either enter or spacebar when testing in Chrome on macOS**

## How to reproduce
1. [Download the Dynamic Accessibility Plugin tool here](https://ibm.ent.box.com/notes/335005856253?s=mhpgjlgss354mcidsv6rvpuu0bxlmfm4)
2. Run the extension using the August 2018 ruleset

### PR DESCRIPTION
**Pull request by dakahn**
Closes #1351

- adds semantic form wrapper to file-uploader 
- removes button role from label since a label can't be a button aria wise
- fixes a bug where there were two tabbable elements in the file-uploader button (label, input)
- adds keyboard navigation to label element

### CODE REVIEW

**Comment by dakahn** [comment]
I'm addressing the build failure. Will upload a fix for the fix soon.

File: src/components/file-uploader/file-uploader.js [hunk 1]
```diff
@@ -7,6 +7,29 @@ import handles from '../../globals/js/mixins/handles';
...
+/**
+ * We're using a pattern where we style an input's label to look like a button.
+ * Consequently we need to add proper keyboard accessibility to the label in
+ * order to mimic if it were an input button -- also worth noting is we hide the
+ * actual input from tab order in the markup.
+ */
+if (document.readyState === 'loading') {
+  document.addEventListener('DOMContentLoaded', () => {
+    const uploadLabel = document.getElementById('input-button');
+    const uploadInput = document.getElementById('your-file-importer-id-here');
```
Comment (asudoh) [inline src/components/file-uploader/file-uploader.js]: Seems that the code here needs to be implemented in `FileUploader` vanilla class - Sent a private note.

**Comment by joshblack** [comment]
bump @dakahn!

**Comment by joshblack** [comment]
Let me know if you want to finish this off or close it and work on it later 👍

**Comment by dakahn** [comment]
I'm gonna close this up for now and revisit the work later. 🙇🏼‍♂️

**Pull request closed without merging**